### ISSUE TITLE
Need ability to orphan a post, move it to a new circle

### ISSUE THREAD

**Issue by hesco** [Phuks-co/throat#294]
If I may express this request as user stories: 
     https://ovarit.com/o/Circles/17041/cant-mods-move-a-wrong-circle-post
     https://ovarit.com/o/Suggestions/16057/ability-for-moderators-to-move-posts-to-correct-circle 

As a circle moderator, when I encounter in the queue of new posts to the circle I moderate, a post or thread which is not suitable to circle or otherwise complaint with its rules and expectations, I want to be able to orphan a post, with a note of its unsuitability for my circle, so that it is only accessible to its original author.

As the author of a post, when I log in, I want a menu item which provides me access to orphaned posts.

As the author of a post, when I am managing an orphaned post, I want a mechanism for attaching that post to a new circle which has not previously rejected it, or of submitting an appeal to the moderator for a circle which has rejected it, including an opportunity to make my case that the post does belong in the circle from which it has been rejected.

As the author of a comment on an orphaned post, I want to receive a private message that the post I have commented on has been orphaned.

As the author of a comment on an orphaned post which has been attached to a new circle, I want to receive a private message that the post I have commented on has been moved to a new circle, and to be provided the option to edit or delete my comment.

As a circle moderator, when I log in, I want a mechanism for reviewing appeals from post authors related to posts which have previously been rejected by myself or another moderator of my circle; and which provides me with a mechanism to accept or deny that appeal with or without a comment attached to the action, with that comment being sent to the private mail of the post author who submitted the appeal.

**Comment by hesco** [comment]
See also:  
https://gitlab.com/feminist-conspiracy/throat/-/issues/120

**Comment by Polsaker** [comment]
Alternative, lazy option: Define a site-wide random/"whatever" sub and allow mods to kick off-topic posts to that sub

**Comment by globalistas** [comment]
Name suggestion for the sub: The Purgatory

**Comment by BoringBuffalo** [comment]
I'm afraid this isn't a feature we want for the reasons I went into here: https://gitlab.com/feminist-conspiracy/throat/-/issues/120

So if it's added, I hope it's added with a config to turn it off.

**Comment by Polsaker** [comment]
Yeah you'll have similar concerns on most sites, that's why I suggested only allowing moving to a pre-determined catch-all sub
